web-console/3-project-details/deployment-details.md

```diff
@@ -0,0 +1,42 @@
+# Deployments: Details Page
+
+- Selecting a deployment from the Deployments landing page or from the Overview will bring the user to the Deployment Details page.
+- The details page for Deployments will include four tabs: History, Configuration, Environment, Events.
+
+## History Tab
+No change at this time.
+
+## Configuration Tab
+No change at this time.
+
+## Environment Tab
+ ![template](img/v2-EnvVar-05.png)
+- This tab will now consist of two parts, (1) the "Variables" section that already existed and (2) the "Environment From" section that is new to the page.
+- These grey (#757575) subheadings will be used to separate the two sections.
+- Once any changes have been made in either area, the Save button should become enabled as a primary action button (in blue).
+
+#### Variables
+- This top section lets the user create new environment variables or create variables by picking one specific key from a secret or config map.
+- Order is important and users can sort variables within this section.
+- The two action links should be called "Add Value" and "Add Value from a Resource."
+
+#### Environment from
+
+- ![template](img/v2-EnvVar-help.png)
+- There should be a Field Level Help icon to the right of this section header.
+  - Clicking the field level help will display the following text: All keys from these config maps and secrets will be added as environment variables.
+  - See the [Field Level Help](http://www.patternfly.org/pattern-library/forms-and-controls/field-level-help/) pattern on PatternFly for guidance on implementation.
+- The column label should be "Secret/ Config Map," and should match the visual treatment of the "Name" and "Value" labels in the section above.
+
+ ![template](img/v2-EnvVar-empty.png)
+- The empty state should include one empty row in each of the sections.
+- The action at the bottom of this section is called "Add ALL Values from a Resource" with the ALL piece in bold font. Clicking this action will add another row below.
+
+ ![template](img/v2-EnvVar-drop.png)
+- Each row will contain a dropdown allowing users to select a secret or config map from the list (using the same dropdown that appears in the top section)
+- Order is important here as well and users can sort variables within this section.
+- Once changes have been saved, a "View Secret" link will also appear to the right of each row allowing users to jump directly to the secret details page in the UI.
```

Comment:
We could definitely explore it. There are some links that take you to another place to do an action, so there are a few different cases.

That's a good point about the "view Secret" link in the overview. Back when you could bind to another deployment, the name of the deployment linked to the deployment, and the "view secret" action was necessary to link to the secret. Probably not needed now. We should open an issue.